src/platform/monitoring/external-services/reducer.js

```diff
@@ -18,8 +19,8 @@ export default function externalServiceStatuses(state = INITIAL_STATE, action) {
       return { ...state, loading: false };
 
     case FETCH_BACKEND_STATUSES_SUCCESS: {
-      const { statuses } = action.data.attributes;
-      return { ...state, loading: false, statuses };
+      const { statuses, maintenanceWindows } = action.data.attributes;
```

Comment:
Hmm.. can you double check this actually comes back as `maintenanceWindows` in camel-case? Usually vets-api returns everything in snake-case.